wcfsetup/install/files/acp/style/layout.scss

```diff
@@ -909,3 +909,26 @@ html[data-color-scheme="dark"] {
 .woltlabNewsfeed--loading .woltlabNewsfeed__iframe {
 	visibility: hidden;
 }
+
+.acpDashboard {
+	display: grid;
+	grid-template-columns: repeat(3, 1fr);
+	gap: 20px;
+	grid-auto-rows: minmax(300px, auto);
+	margin-top: 40px;
+}
+
+.acpDashboardBox {
+	background-color: var(--wcfContentContainerBackground);
+	border-radius: var(--wcfBorderRadius);
+	box-shadow:
+		0 0 3px rgba(0, 0, 0, 0.12),
+		0 1px 2px rgba(0, 0, 0, 0.24);
+	padding: 20px;
+}
+
+.acpDashboardBox__title {
+	margin-bottom: 20px;
```

Comment:
I would rather set `.acpDashboardBox` to `display: flex` with `flex-direction: column` and `row-gap: 20px`. It is a bit more verbose, but avoids the use of margins and moves the layout to the top-most block.